static/js/util.js

```diff
@@ -39,24 +39,16 @@ function wrapSubstrings(text, words, prefix, suffix) {
   return result;
 }
 
-function select2Highlighter($container) {
-  let $searchField;
-
-  // The search field is only available once the dropdown is rendered, so we
-  // set it in the immediate next event loop:
-  setTimeout(function() {
-    $searchField = $container.next('.select2-container').find('.select2-search__field');
-  }, 1);
-
-  return function(state) {
-    let query = $.trim($searchField.val());
-    let text = wrapSubstrings(
-      state.text,
-      query.split(/\s+/),
-      '<span class="select2-highlight">',
-      '</span>',
-    );
-
-    return $('<div>' + text + '</div>');
-  };
+function select2Highlighter(state) {
+  $searchField = $('.select2-container--open .select2-search__field');
```

Comment:
The original "smart" way to find the search field once doesn't work out, so we'll just fetch the DOM element for every result. It's not ideal, but it works.